.github/workflows/build.yml

```diff
@@ -1,33 +1,45 @@
 name: CI
+
 on:
   push:
     branches:
       - "**"
   pull_request:
     branches:
       - "**"
+
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        deno-version: [1.x]
```

Comment:
Is it possible to change this to use `.denov`?

Reply:
Hmm I don't think so because `cat .denov` needs to be run by `run` command